### ISSUE TITLE
Filipino translations page required.

### ISSUE THREAD

**Issue by kakashi215** [EbookFoundation/free-programming-books#4637]
How to add Filipino translations? Wanted to add one.
Since there is no file for that at the moment.
I can create one but what to name the file - free-programming-books-??.md

Regards.

**Comment by kakashi215** [comment]
@eshellman What should be done for this new addition?

**Comment by kakashi215** [comment]
So there is no 2 character code for Filipino (No 639-1 code) but a 3 character code (639-2) for Filipino as "**fil**".
Requesting to allow me to make the aforementioned changes.

**Comment by eshellman** [comment]
do you mean tagalog? that would be `tl`

**Comment by kakashi215** [comment]
Am talking about Filipino language.
For eg. 
For French, we got fr. (free-programming-books-**fr**.md)
For Russian we got ru. (free-programming-books-**ru**.md)

How to name the Filipino document?

**Comment by SethiPandi** [comment]
TL;DR: I'm not a linguist, but I think...
* Tagalog can be used for the language code if we're keen on using ISO 639-1, or 2 character long language codes. 
* Alternatively, as Filipino is officially a language, it could be nice to use the ISO 639-2 or ISO 639-3, but if this happens it might be worth considering if existing files should be changed to use ISO 639-3 character long language codes as well for consistency, for example, `eng` instead of `en`.

I checked [Weblate](https://weblate.org/en/) which is a cloud translation platform, they appear to use ISO 639-1, unless the language doesn't have one, then it uses 639-3 instead, for example for Filipino.

---

Since the repository currently has all languages with 2 characters, it looks like we're using ISO 639-1 language codes.

I just gave it a peek, I don't see an ISO 639-1 code for [Filipino](https://en.wikipedia.org/wiki/Filipino_language). :thinking:

The list of ISO 639-1 codes on Wikipedia note under the language "Tagalog":
> Note: [Filipino](https://en.wikipedia.org/wiki/Filipino_language) (Pilipino) has the code [fil] 
> \- https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes

It seems Filipino and Tagalog are separate languages on paper, however they're practically the same if not synonyms?

> While the official view (shared by the government, the Komisyon ng Wikang Filipino, and a number of educators) is that Filipino and Tagalog are considered separate languages, in practical terms, Filipino may be considered the official name of Tagalog, or even a synonym of it.
> \- https://en.wikipedia.org/wiki/Filipino_language#Filipino_versus_Tagalog

Filipino does have its own ISO 639-2 / ISO 639-3 code though of `fil` as @kakashi215 has pointed out.

The following is a list of ISO 639-2 codes on Wikipedia:  
https://en.wikipedia.org/wiki/List_of_ISO_639-2_codes

**Comment by SethiPandi** [comment]
As an addition, regardless of the conclusion of this.

May I suggest whatever standard is being used for the language, that it gets documented in the contribution guide?

**Comment by eshellman** [comment]
yes, we use iso language codes. so, for example, german is "de" and spanish is 'es'

**Comment by SethiPandi** [comment]
What I was pointing out though is that it looks like we're using ISO 639-**1** though, but Filipino which is an official language, doesn't have a ISO 639-**1** code; only a ISO 639-**2** and ISO 639-**3** code.

So, in other words, should the repository use an ISO 639-**3** code (3 characters) if a language doesn't have an ISO 639-**1** code (2 characters)?

**Comment by kakashi215** [comment]
I think we should use 639-3 codes. It is important to officially support all languages.
Using closest similar language (like Tagalog for filipino) isn't a scalable approach.
Hence I'd like to work with 639-3 codes as well.

**Comment by eshellman** [comment]
I'll defer to a Filipino speaker on this. either fil of tl is ok, but I don't want Filipinos complaining about the choice! https://en.wikipedia.org/wiki/Tagalog_language#Controversy

**Comment by davorpa** [comment]
I copy some feedback made by natives

_Originally posted by @genepride, @petemadis, @cjjp-sushi in https://github.com/EbookFoundation/free-programming-books/issues/6504#issuecomment-956519688_
> If you label the language of the Filipinos you can just use the .fil (Filipino). Tagalog is where the Filipino language was derived from, lets just say the Tagalog language is the old version and the Filipino language is the new version.
> 
> In a sense, the Filipino language is just an upgraded 2.0 version of Tagalog.
> 
> Filipino is the national language of the Philippines. Are "Tagalog," "Pilipino" and "Filipino" different languages? No, they are mutually intelligible varieties, and therefore belong to one language.
